cli/lsp/config.rs

```diff
@@ -1534,17 +1534,11 @@ impl ConfigData {
           import_map_url.clone(),
           ConfigWatchedFileType::ImportMap,
         );
-        // spawn due to the lsp's `Send` requirement
-        let fetch_result =
-          deno_core::unsync::spawn({
```

Comment:
removing all these spawn calls seems like a win

Reply:
That's from #28035. We're likely not going with this approach (re the raw pointers)